examples/tpm2-ekcert/main.go

```diff
@@ -51,6 +47,22 @@ var (
 	}
 )
 
+// Auth Policy calculations in Credential_Profile_EK_V2.0, section 2.1.5.3, "authPolicy"
+func defaultAuthPolicy() []byte {
+	previousDigest := make([]byte, 32)            // All zero buffer
+	policySecretCommandCode := uint32(0x00000151) // TPM__CC_PolicySecret
+	name := uint32(0x4000000B)                    // TPM_RH_ENDORSEMENT
```

Comment:
Also this to `constants.go`

Reply:
Fixed, in fact this already exists as `tpm2.HandleEndorsement`